ra2ce/network/network_simplification/snkit_network_wrapper.py

```diff
@@ -47,6 +47,14 @@ class SnkitNetworkWrapper:
     edge_from_id_column_name: str = "from_id"
     edge_to_id_column_name: str = "to_id"
 
+    def __init__(self, snkit_network: Network, node_id_column_name: str = "id",
+                 edge_from_id_column_name: str = "from_id", edge_to_id_column_name: str = "to_id") -> None:
+        self.snkit_network = snkit_network
+        self.node_id_column_name = node_id_column_name
+        self.edge_from_id_column_name = edge_from_id_column_name
+        self.edge_to_id_column_name = edge_to_id_column_name
+        self.networkx_graph: NxGraph = self.to_networkx()
```

Comment:
why would you do this? When are you using this property that needs to be executed right away? This will take a toll in computation.

Reply:
I do not understand what you mean here and how it should be resolved, can you give me an example or so?